pennylane_lightning/lightning_gpu/lightning_gpu.py

```diff
@@ -384,4 +403,27 @@ def simulate(
 
         Note that this function can return measurements for non-commuting observables simultaneously.
         """
-        return 0
+        if circuit.shots and (any(isinstance(op, MidMeasureMP) for op in circuit.operations)):
```

Comment:
This branch may not be necessary since we do not support `MidMeasureMP` natively.

Reply:
Do you think that this is enough? :thinking: 
``` python
if circuit.shots and (any(isinstance(op, MidMeasureMP) for op in circuit.operations)):
            raise qml.DeviceError("LightningGPU does not support Mid-circuit measurements.")
```

Reply:
We can raise just in case, but this should be caught by `preprocess`.